### PR TITLE
data:Kordedekehine

### PR DESCRIPTION
**Pull request by Kordedekehine**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4352"></a>

### CODE REVIEW

File: Kordedekehine.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "name": "Salami Korede",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software,Open Source Enthusiast and Content Creator",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/akehindekorede",
+      "icon": "FaTwitter"
+    }
```
Comment (ChinmayMhatre) [inline Kordedekehine.json:11]:
```suggestion
    },
```

File: Kordedekehine.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Kordedekehine",
+      "icon": "FaGithub"
+    }
```
Comment (ChinmayMhatre) [inline Kordedekehine.json:16]:
```suggestion
    },
```

File: Kordedekehine.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/salami-kehinde-korede-bb85281ab/",
+      "icon": "FaLinkedIn"
+    }
```
Comment (ChinmayMhatre) [inline Kordedekehine.json:21]:
```suggestion
    },
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Hey @Kordedekehine , Profile looks great. A few minor changes.

-  Please add the suggestions in the json file.
- Please move the json file to the `data` directory.


Thank you!

**Review by Hrishi5111998 — COMMENTED** [review]
hey @Kordedekehine, please update **icon** value to **FaLinkedin** rather than FaLinkedIn. thank you!

**Comment by Kordedekehine** [comment]
> 

I have made the changes now. Thank you.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please move your file to `data` directory. You need to rename your [file here](https://github.com/Kordedekehine/LinkFree/edit/Kordedekehine-patch-1/Kordedekehine.json) to exact `data/Kordedekehine.json`.

- Change `Kordedekehine.json` to `data/Kordedekehine.json` here. 

Let us know if you need any help with this! :)

image

**Review by rupali-codes — COMMENTED** [review]
lgtm

**Comment by Kordedekehine** [comment]
> Please move your file to `data` directory. You need to rename your [file here](https://github.com/Kordedekehine/LinkFree/edit/Kordedekehine-patch-1/Kordedekehine.json) to exact `data/Kordedekehine.json`.
> 
> * Change `Kordedekehine.json` to `data/Kordedekehine.json` here.
> 
> Let us know if you need any help with this! :)
> 
> image

Fixed now

**Comment by kumarsonsoff3** [comment]
It seems to be a Duplicate of #4388, which is being merged recently. So closing this one. 

--------------------------
You can access your profile here https://linkfree.eddiehub.io/Kordedekehine. 
For any changes you need to make in your profile, you can follow this guide - https://linkfree.eddiehub.io/docs/how-to-guides/editing :)

**Pull request closed without merging**